app/views/api/v3/analytics/user_analytics/_data_stacked_bar_graph.html.erb

```diff
@@ -50,13 +51,39 @@
       </div>
       <div class="p-absolute l-0 w-auto mw-100 p-8px bgc-black br-4px" data-element-type="tooltip-copy">
         <p class="m-0px p-0px ta-left fw-regular fs-14px lh-150 c-white">
-          <%= data["bp_not_controlled_patients"] %> patients with &ge;140/90
+          <%= data["bp_not_controlled_patients"] %> <%= "patient".pluralize(data["bp_not_controlled_patients"]) %> with <%= t("progress_tab.diagnosis_report.diagnosis_thresholds.hypertension_uncontrolled_long") %>
         </p>
       </div>
     </div>
   </div>
   <div
-    class="d-flex ai-center jc-center ml-2px p-16px bgc-red-new btrr-4px bbrr-4px ttf-ease-in-out td-0_25s tp-opacity"
+    class="d-flex ai-center jc-center ml-1px mr-1px p-16px bgc-orange-new ttf-ease-in-out td-0_25s tp-opacity"
+    style="width: <%= visited_but_no_bp_taken %>%;"
+    data-graph-element="stacked-bar"
+  >
+    <p class="m-0px p-0px ta-center fw-bold fs-16px c-white pe-none">
+      <%= data["visited_but_no_bp_taken_rate"] %>%
+    </p>
+    <div
+      class="o-0 pe-none p-absolute l-0 zi-100 d-flex fd-column w-100 bs-tooltip ttf-ease-in-out td-0_25s tp-opacity"
+      data-element-type="tooltip"
+      data-tooltip-index="1"
+      data-tooltip-length="3"
+    >
+      <div
+        class="p-absolute b--8px w-0px h-0px br-8px-transparent bl-8px-transparent bt-8px-black"
+        data-element-type="tooltip-tip"
+      >
+      </div>
+      <div class="p-absolute l-0 w-auto mw-100 p-8px bgc-black br-4px" data-element-type="tooltip-copy">
+        <p class="m-0px p-0px ta-left fw-regular fs-14px lh-150 c-white">
+          <%= data["visited_but_no_bp_taken_patients"] %> <%= "patient".pluralize(data["visited_but_no_bp_taken_patients"]) %> with <%= t("progress_tab.diagnosis_report.diagnosis_thresholds.hypertension_visited_but_no_bp_taken_long") %>
```

Comment:
I think you can use a single I18n string here, instead of breaking it down into three strings. 

Currently both `patient(s)` and `with` will not be internationalized when we need support other languages.

Reply:
Good call @govindkrjoshi will fix :)